desktop/apps/auction2/components/filter_sort/index.js

```diff
@@ -0,0 +1,54 @@
+import { default as React, PropTypes } from 'react';
+import { connect } from 'react-redux'
+import _ from 'underscore'
+import { updateSort } from '../../actions'
+
+function FilterSort({ sortMap, filterParams, updateSortAction }) {
+  const selectedSort = filterParams.sort
+  return (
+    <div className='auction2-filter-sort'>
+      <span className='auction2-filter-sort__label bordered-pulldown-label'>Sort by:</span>
+      <div className='bordered-pulldown'>
+        <a className='bordered-pulldown-toggle' href='#'>
+          <span className='bordered-pulldown-text'>{ sortMap[selectedSort] }</span>
+          <div className='bordered-pulldown-toggle-caret'>
+            <span className='caret'></span>
+          </div>
+        </a>
+        <div className='bordered-pulldown-options'>
+          {
+            _.map(sortMap, (sortName, sort) => {
```

Comment:
I would just make the point that it's a bit of a judgment call. In some cases, Lodash methods actually outperforms native on some browsers, although I'm guessing this isn't the usual case. But either way, the difference is typically negligible. 

If the standard lib really did have parity with Lodash, I'd probably 100% agree with you, because I think calling methods on collections is very intuitive. But since the standard library is still rather incomplete, then it's kind of a matter of style whether to go mixed Lodash standalone functions and native methods, or to consistent Lodash standalone functions. (Or, [Lodash chaining style](https://lodash.com/docs/4.17.4#lodash).)

But one way I often think about such things is, "if performance is negligble, what's best for readability and maintainability?" I'm not trying to favor a particular answer, but just perhaps framing the question.